### ISSUE TITLE
混合Reward 训练

### ISSUE THREAD

**Issue by YujieOuO** [XueZeyue/DanceGRPO#19]
你好，
非常感谢Dancegrpo的开源工作

我在混合训练 hps + clip score的时候，似乎只有hps的点在提升，而 clip score 维持在一个稳定水平大概在0.31左右。
请问这是合理的吗，因为论文上显示哪怕不训练clip score也有0.363，联合训练的话能到0.395。
所以想请教下是Clip 模型没有选对，还是说测试Clip Score需要专门的测试集呢？

<img width="666" alt="Image" src="https://github.com/user-attachments/assets/693b43ac-b0ab-49cf-93a8-610b3e1aa579" />

模型：SD v1.4
Clip Model：apple/DFN5B-CLIP-ViT-H-14-378
计算代码：
`   ## hps score
      outputs = reward_model(image, text)
      image_features, text_features = outputs["image_features"], outputs["text_features"]
      logits_per_image = image_features @ text_features.T
      hps_score = torch.diagonal(logits_per_image)
      eval_hps_rewards.append(hps_score)

       ## clip score
      clip_image_features = clip_model.encode_image(image)
      clip_text_features = clip_model.encode_text(text)
      clip_image_features = F.normalize(clip_image_features, dim=-1)
      clip_text_features = F.normalize(clip_text_features, dim=-1)
      clip_score = clip_image_features @ clip_text_features.T
      eval_clip_rewards.append(clip_score[0])`

**Comment by YujieOuO** [comment]
然后 因为我是在sd上训练的，所以基于sd的混合训练，也是 hps/clip 0.7:1.4 吗

**Comment by XueZeyue** [comment]
1. 是的，我打开这个https://huggingface.co/apple/DFN5B-CLIP-ViT-H-14-378，里面的示例代码是384。。。。咱也没懂，反正就用384跑了
2. 对，0.7:1.4，跑出来图片能去油应该就对了

**Comment by YujieOuO** [comment]
很棒棒

**Comment by YujieOuO** [comment]
这里的Clip Score可视化是不是做了滑动平均，我跑的还是抖得厉害

**Comment by XueZeyue** [comment]
我所有的可视化都有滑动平均的，不然跑出来没法看
